sandbox/modules/extensible-identity/build.gradle

```diff
@@ -0,0 +1,22 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+opensearchplugin {
+  description 'Extensible Plugin for identity features in OpenSearch.'
+  classname 'org.opensearch.extensible.identity.ExtensibleIdentityPlugin'
+  name 'extensible-identity'
+  licenseFile rootProject.file('licenses/APACHE-LICENSE-2.0.txt')
+  noticeFile rootProject.file('NOTICE.txt')
+}
+
+dependencies {
+  implementation project(':sandbox:libs:opensearch-authn')
+}
+
+test {
+  systemProperty 'opensearch.set.netty.runtime.available.processors', 'false'
```

Comment:
When this gets merged will it have any impacts outside of our use or is it a relatively minor difference? I know it sets the number of processors and I think that setting it to false will make it not restrict the netty thread pool size. Is this intended?